packages/langium/src/grammar/langium-grammar.langium

```diff
@@ -16,13 +16,21 @@ entry Grammar:
 Interface:
     'interface' name=ID
     ('extends' superTypes+=[AbstractType:ID] (',' superTypes+=[AbstractType:ID])*)?
-    SchemaType;
-
-fragment SchemaType:
-    '{' attributes+=TypeAttribute* '}' ';'?;
+    '{' 
+        attributes+=TypeAttribute* 
+    '}' 
+';'?;
 
 TypeAttribute:
-    name=FeatureName (isOptional?='?')? ':' type=TypeDefinition ';'?;
+    name=FeatureName (isOptional?='?')? ':' type=TypeDefinition ('=' defaultValue=ValueLiteral)? ';'?;
```

Comment:
Is it possible to support default for arrays of primitive types? Or do you plan to only support primitive types?

My use case that I want to parse `yaml` style code in `mermaid` and some properties have array default values.

```ts
{
  ...,
  timeline: {
    ...,
    sectionColours: ['#fff']
  }
}
```

Or should I just override the parser and set default values if it's empty array? It would take more manual work and time since we need to figure out each default and update in both schema and setter.

Reply:
Sure, that's a good idea :)

I'll add that to the PR when I have time. I'm currently quite busy.

Reply:
Added the feature to the PR 👍